### ISSUE TITLE
Not showing thumbnail with custom UI

### ISSUE THREAD

**Issue by Harze2k** [TheAMM/mpv_thumbnail_script#73]
Hi!
I'm using the excellent Mpv UI script found here: https://github.com/darsain/uosc
Its addictive to tinker with and a nice extra feature to Mpv.
Is there a way to make it so that the thumbs show when using this UI script? I spoke with @darsain and he said it would be way easier to implement this possibility in the thumbnail script rather then his UI. 

Just putting it out there, would be amazing if the could work together 👍

**Comment by hooke007** [comment]
Yes I tried to do that work once. But I finally gave it up. It will spend too much time and too high usage generating the thumbs. The way it works actually look like re-encoding the video to output the fixed amounts of pictures rather than the real thumbnailer-viewer as other players (https://github.com/TheAMM/mpv_thumbnail_script/issues/59#issuecomment-633790523).
The script is really wonderful, however maybe not friendly to most computers.
